src/cgame/cg_rocket_datasource.cpp

```diff
@@ -427,7 +418,7 @@ void CG_Rocket_CleanUpServerList( const char *table )
 	int j;
 	int netSrc = CG_StringToNetSource( table );
 
-	for ( i = AS_LOCAL; i <= AS_FAVORITES; ++i )
+	for ( i = AS_LOCAL; i <= AS_NUM_TYPES; ++i )
```

Comment:
Is this on purpose or did you mean this instead?
```suggestion
	for ( i = AS_LOCAL; i < AS_NUM_TYPES; ++i )
```

Reply:
Thanks, fixed